src/components/BankTransactionList/BankTransactionListItem.tsx

```diff
@@ -69,6 +72,9 @@ export const BankTransactionListItem = ({
 
   const [showComponent, setShowComponent] = useState(false)
 
+  const bookkeepingStatus = useEffectiveBookkeepingStatus()
+  const categorizationEnabled = isCategorizationEnabledForStatus(bookkeepingStatus)
```

Comment:
In this file, I disable buttons and category select when bookkeeping active, and therefore categorization is disabled